CHANGELOG-3.6.md

```diff
@@ -15,6 +15,7 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0).
 ### etcdctl v3
 
 - Add command to generate [shell completion](https://github.com/etcd-io/etcd/pull/13133).
+- Show db size in use when print endpoint status [#13639](https://github.com/etcd-io/etcd/pull/13639)
```

Comment:
Can you please change it the way it's formatted in the above line i.e. hyperlink [Show db size in use] instead of the actual PR number. Thanks!

Reply:
fixed！
Thanks！